Python/Instagram-Liker-Bot/READme.md

```diff
@@ -0,0 +1,24 @@
+# Use of Script :
```

Comment:
Please follow the [template](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md). The README needs a lot of alterations.